app/aggregates/deciding/commands/create_draft.rb

```diff
@@ -2,10 +2,11 @@ module Deciding
   class CreateDraft < Command
     attribute :application_id, Types::Uuid
     attribute :user_id, Types::Uuid
+    attribute :reference, Types::Integer
 
     def call
       with_decision do |decision|
-        decision.create_draft(user_id:, application_id:)
+        decision.create_draft(user_id:, application_id:, reference:)
```

Comment:
Including reference in all deciding events simplifies linking to streams based on reference for submission history.